ui/app/AppLayouts/Chat/controls/community/ThumbnailsDropdownContent.qml

```diff
@@ -0,0 +1,111 @@
+import QtQuick 2.0
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.14
+
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+
+StatusScrollView {
+    id: root
+
+    property string title: ""
+    property url titleImage: ""
+    property string subtitle: ""
+    property ListModel model
+    property int maxHeight: 381 // default by design
+
+    signal itemClicked(var key, string name, url iconSource)
```

Comment:
why var and not string ?

Reply:
Since we still don't have the backend for that, I've thought to use `var` instead to have it more opened. WDYT?